rr_endline_detection/include/endline_detection.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef ENDLINE_DETECTION_HPP
```

Comment:
add a file descriptor matching the one in our README

Reply:
done